0x/Assets/ERC20Asset.cs

```diff
@@ -0,0 +1,53 @@
+﻿using Nethereum.Util;
+using System;
+using System.Linq;
+using ZeroX.Utilities;
+
+namespace ZeroX.Assets
+{
+
+    /// <summary>
+    /// ERC20 asset
+    /// </summary>
+    public class ERC20Asset : TokenAsset
+    {
+
+        private static readonly byte[] _erc20AssetHeader = new byte[] { 0xf4, 0x72, 0x61, 0xb0 };
```

Comment:
I would put this in constants file.

Reply:
I would not, currently this is only used in this class.